rbd.c

```diff
@@ -885,6 +889,7 @@ static int tcmu_rbd_flush(struct tcmu_device *dev, struct tcmulib_cmd *cmd)
 
 	aio_cb->dev = dev;
 	aio_cb->tcmulib_cmd = cmd;
+	aio_cb->read = false;
 	aio_cb->bounce_buffer = NULL;
 
```

Comment:
tcmu_rbd_aio_writesame also needs to get modified right?

Reply:
I modified it。Thanks！